pkgs/sdk/server/src/Internal/FDv2DataSources/Changeset.cs

```diff
@@ -0,0 +1,139 @@
+using System;
+using System.Collections.Immutable;
+using System.Text.Json;
+
+namespace LaunchDarkly.Sdk.Server.Internal.FDv2DataSources
+{
+    /// <summary>
+    /// Represents the type of change operation.
+    /// </summary>
+    internal enum ChangeType
```

Comment:
The individual change types, and the type of the changeset itself, are independent from the protocol types.